lib/Runtime/Library/ConcatString.h

```diff
@@ -9,18 +9,30 @@ namespace Js
     class LiteralStringWithPropertyStringPtr : public LiteralString
     {
     private:
+        // TODO: Can we make PropertyString && LiteralStringWithPropertyStringPtr share the string buffer?
         PropertyString * propertyString;
```

Comment:
@MikeHolman do you know why this isn't a `Field`? The CI seems to be complaining about it now, and I'm inclined to agree with it.

Reply:
Yeah, it should be one.